src/util/testUtils.js

```diff
@@ -108,3 +109,20 @@ export function testCreateStore(createStoreFn) {
 		basicStore.dispatch({ type: 'dummy' });
 	});
 }
+
+/**
+ * Useful function when generating fake state objects
+ * @param {String} ref Reference to use in object creation
+ * @param {Object} value value to use in object creation
+ * @return {Object} mock object
+ */
+export const generateMockState = (ref, value = { id: '🦄' }) => ({
+	api: {
+		...MOCK_APP_STATE.api,
+		[ref]: {
+			ref: ref,
+			meta: { statusCode: 204 },
```

Comment:
applying defaults for multiple arguments gets a little awkward when you want to override one default but not another, so I think I'll need to change the signature of this function from `(ref, value?)` to `({ ref, value?, meta? })` - I'll make a note to update that breaking change in mup-web as well.